.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Can you remove, I think these setups tend to be quite subjective preference, so would want to leave that up to the individual user.

Reply:
Makes sense 👍